src/pages/ValidateLoginPage/index.website.tsx

```diff
@@ -71,13 +73,14 @@ function ValidateLoginPage({
             {autoAuthState === CONST.AUTO_AUTH_STATE.FAILED && <ExpiredValidateCodeModal />}
             {autoAuthState === CONST.AUTO_AUTH_STATE.JUST_SIGNED_IN && is2FARequired && !isSignedIn && <JustSignedInModal is2FARequired />}
             {autoAuthState === CONST.AUTO_AUTH_STATE.JUST_SIGNED_IN && isSignedIn && !exitTo && <JustSignedInModal is2FARequired={false} />}
-            {autoAuthState === CONST.AUTO_AUTH_STATE.NOT_STARTED && !exitTo && (
+            {/* If session.autoAuthState isn't available yet, we use shouldStartSignInWithValidateCode to conditionally render the component instead of local autoAuthState which contains a default value of NOT_STARTED */}
```

Comment:
It's commented here or which part do you mean?

Reply:
Hmm, I think that would be tricky since I'm not the one who added the condition and I agree the condition is confusing, so I don't know for example why we need to check for `!exitTo`.
https://github.com/Expensify/App/blob/e8ae3c5acedf0e6788dc574c7b6f3043ca37092a/src/pages/ValidateLoginPage/index.website.tsx#L73